decidim-core/db/migrate/20180810092428_move_organization_fields_to_hero_content_block.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class MoveOrganizationFieldsToHeroContentBlock < ActiveRecord::Migration[5.2]
+  def change
+    Decidim::Organization.find_each do |organization|
+      content_block = Decidim::ContentBlock.find_by(organization: organization, scope: :homepage, manifest_name: :hero)
+      settings = {}
+      welcome_text = organization.welcome_text
+      settings = welcome_text.inject(settings) { |acc, (k, v)| acc.update("welcome_text_#{k}" => v) }
+
+      content_block.settings = settings
+      content_block.images_container.background_image = organization.homepage_image.file
```

Comment:
Have you tried this with "real" data?

Reply:
I tried it with the development app, which had this data. Does this count as "real data"?